README.md

```diff
@@ -103,7 +102,7 @@ If you are compiling Armadillo by hand, ensure that LAPACK and BLAS are enabled.
 ### 4. Building mlpack from source
 
 This section discusses how to build mlpack from source.  However, mlpack is in
-the repositories of many Linux distributions and so it may be easier to use the
+the repositories of many Linux distributions so it may be easier to use the
```

Comment:
The `and so` is fine too, but I don't mind changing it.  I'd suggest adding a comma: `many Linux distribution, so`.